packages/Dockerfile.debian

```diff
@@ -1,36 +1,64 @@
-ARG DOSBOX_VERS=v0.00.0
-ARG BASE_IMG=ghcr.io/dosbox-staging/debian-cross-base
-ARG DEB_VERS=10
-
-FROM ${BASE_IMG}:${DEB_VERS}-${DOSBOX_VERS} as final
-ARG DPKG_ARCH=arm64
+ARG DEB_VERS
+ARG DPKG_ARCH=""
 ARG DOSBOX_APT_LIST=/debian-apt-list.txt
+FROM debian:${DEB_VERS}-slim
 ARG DEB_VERS
+ARG DPKG_ARCH
+ARG DOSBOX_APT_LIST
+
+# Enable cross compile architecture. Abort if not set correctly
+RUN case "$DPKG_ARCH" in \
+        arm64|armhf|ppc64el) dpkg --add-architecture $DPKG_ARCH ;; \
+        *)  echo "DPKG_ARCH not set!" && exit 1 ;; \
+    esac
 
-ENV MESON_CROSS_FILE=/usr/local/share/meson/cross/dosbox-cross
+ENV DEBIAN_FRONTEND=noninteractive
+# Install common base package
+RUN apt-get update && apt-get -y install --no-install-recommends \
```

Comment:
Do we want to also do an `apt-get upgrade` here following the `apt-get update`?  Update only downloads the latest list of packages.  Upgrade actually upgrades the packages.

Reply:
The base Debian image this builds on is already updated regularly. I see no reason to upgrade packages again, which just increases the final image size.